lib/github-model.js

```
@@ -0,0 +1,82 @@
+const token = process.env.GITHUB_MODEL_TOKEN || require('../.secrets/gh-model').token;
+
+if (!token) {
+  console.error('GITHUB_MODEL_TOKEN environment variable is not set');
+  process.exit(1);
+}
+const config = require('../config/github-model');
+const yaml = require('js-yaml');
+
+// Create a reusable axios instance with keepAlive enabled
+const https = require('https');
+const https = require('https');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'https'. Consider to delete it or rename it.

'https' is defined here:


(lint/suspicious/noRedeclare)